rocAL/rocAL/source/rali_api_meta_data.cpp

```diff
@@ -326,3 +343,37 @@ RALI_API_CALL raliCreateTextCifar10LabelReader(RaliContext p_context, const char
 
 }
 
+void RALI_API_CALL raliBoxEncoder(RaliContext p_context, std::vector<float>& anchors, float criteria,
+                                  std::vector<float> &means, std::vector<float> &stds, bool offset, float scale)
+{
+    if (!p_context)
+        THROW("Invalid rali context passed to raliBoxEncoder")
+    auto context = static_cast<Context *>(p_context);
+    context->master_graph->box_encoder(anchors, criteria, means, stds, offset, scale);
+}
+
+void 
+RALI_API_CALL raliCopyEncodedBoxesAndLables(RaliContext p_context, float* boxes_buf, int* labels_buf)
+{
+    if (!p_context)
+        THROW("Invalid rali context passed to raliCopyEncodedBoxesAndLables")
+    auto context = static_cast<Context *>(p_context);
+    auto meta_data = context->master_graph->meta_data();
+    size_t meta_data_batch_size = meta_data.second->get_bb_labels_batch().size();
+    if (context->user_batch_size() != meta_data_batch_size)
+        THROW("meta data batch size is wrong " + TOSTR(meta_data_batch_size) + " != " + TOSTR(context->user_batch_size()))
+    if (!meta_data.second)
+    {
+        WRN("No encoded labels and bounding boxes has been loaded for this output image")
+        return;
+    }
+    // copy labels buffer & bboxes buffer
+    for (unsigned i = 0; i < meta_data_batch_size; i++)
```

Comment:
investigate OMP here. But make sure to use temp pointers for lables_buf and boxes_buf